tests/integration/test_catalogs.py

```diff
@@ -83,6 +91,32 @@ async def test_catalogs_serializers(client: _TestClient, mocker: MockFixture) ->
     assert 406 == response.status, "not/acceptable failed"
 
 
+@pytest.mark.integration
+async def test_catalogs_content_negotiation(
+    client: _TestClient, mocker: MockFixture
+) -> None:
+    """Should return 200 and correct content-type."""
+    # Set up the mock
+    mocker.patch(
+        "SPARQLWrapper.SPARQLWrapper.queryAndConvert",
+        return_value=_mock_query_and_convert_result(),
+    )
+
+    headers = MultiDict([(hdrs.ACCEPT, "text/turtle, application/ld+json")])
+    response = await client.get("/catalogs", headers=headers)
+    assert 200 == response.status
+    assert (
+        "text/turtle; charset=utf-8" == response.headers["Content-Type"]
+    ), " Content-Type in response-header should be text/turtle."
+
+    headers = MultiDict([(hdrs.ACCEPT, "application/ld+json, text/turtle")])
+    response = await client.get("/catalogs", headers=headers)
+    assert 200 == response.status
+    assert (
+        "application/ld+json; charset=utf-8" == response.headers["Content-Type"]
+    ), " Content-Type in response-header should be application/ld+json."
+
```

Comment:
Kan du legge inn denne testen?

```suggestion

    headers = MultiDict([(hdrs.ACCEPT, "*/*, text/plain")])
    response = await client.get("/catalogs", headers=headers)
    assert 200 == response.status
    assert (
        "text/turtle; charset=utf-8" == response.headers["Content-Type"]
    ), " Content-Type in response-header should be text/plain."
```

Reply:
Bra spotta. Feila. Antagelegvis også grunnen til at request frå nettleser feiler. Den har følgande header:
`text/html,application/xhtml+xml,application/xml;q=0.9,image/avif,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3;q=0.9`